src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+    private static Map<String,Integer> expectedStorage;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitDaoImpl();
+        expectedStorage = new HashMap<>();
+        expectedStorage.put("banana",555);
+        expectedStorage.put("apple",111);
+    }
+
+    @Test
+    public void put_IsOk() {
+        fruitDao.put("apple",111);
+        Assert.assertEquals(expectedStorage.get("apple"),fruitDao.get("apple"));
+    }
+
+    @Test
+    public void get_IsOk() {
+        fruitDao.put("banana",555);
```

Comment:
it would be better to use `FruitStorage.storage.put` here, because maybe `fruitDao.put` works incorrect